bom/pom.xml

```diff
@@ -1144,6 +1144,12 @@
                 <artifactId>helidon-integrations-neo4j-metrics</artifactId>
                 <version>${helidon.version}</version>
             </dependency>
+            <!-- Bean Validation -->
+            <dependency>
+                <groupId>io.helidon.microprofile.beanvalidation</groupId>
+                <artifactId>helidon-microprofile-bean-validation</artifactId>
```

Comment:
I would add dash to the `groupId` so we keep the convention(`<groupId>io.helidon.microprofile.reactive-streams</groupId>`)
```xml
                <groupId>io.helidon.microprofile.bean-validation</groupId>
                <artifactId>helidon-microprofile-bean-validation</artifactId>
```

Reply:
Applied!